src/commcare_cloud/manage_commcare_cloud/list_vault_keys.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+from itertools import chain
+
+from commcare_cloud.commands.command_base import CommandBase
+from commcare_cloud.environment.main import get_environment
+from commcare_cloud.environment.paths import get_available_envs
+
+
+class ListVaultKeys(CommandBase):
+    command = 'list-vault-keys'
+    help = """
+    Audit the structure of vault.yml files.
+
+    A password will be required for each encrypted vault file in the env.
+    """
+
+    def run(self, args, unknown_args):
+        envs = sorted(get_available_envs(exclude_symlinks=True))
+        var_keys = {}
+        for env in envs:
+            print('[{}] '.format(env), end='')
+            environment = get_environment(env)
+            var_keys[env] = set(recursive_keys(environment.get_vault_variables()))
+
+        for env in envs:
+            print('\t{}'.format(env), end='')
```

Comment:
Why do you have this print statement and the print in line 21?

Reply:
...whereas this line is to print the header row of the table this command is meant to output

Reply:
oh got it. How does `env` have a different value in line 21 vs 26?

Reply:
Ah, the header row is a header of env names, I thought it was something else hence the confusion. Thanks!